docs/commands/New-PesterConfiguration.mdx

```diff
@@ -53,36 +53,48 @@ Run:
   TestExtension: Filter used to identify test files.
   Default value: '.Tests.ps1'
 
-  Exit: Exit with non-zero exit code when the test run fails. When used together with Throw, throwing an exception is preferred.
+  Exit: Exit with non-zero exit code when the test run fails.
+When used together with Throw, throwing an exception is preferred.
```

Comment:
What's causing these linebreaks? Max line limit in platyPS, Alt3-module or editorconfig?

Reply:
Caused by the Alt3-module regex responsible for recognizing markdown headers. Good thing this one came up, the regex was flaky as **. Solved.